src/Parsers/ASTDeleteQuery.h

```diff
@@ -19,6 +19,11 @@ class ASTDeleteQuery : public ASTQueryWithTableAndOutput, public ASTQueryWithOnC
         return removeOnCluster<ASTDeleteQuery>(clone(), params.default_database);
     }
 
+    /** Used in DROP PARTITION, ATTACH PARTITION FROM, UPDATE, DELETE, DELETE FROM queries.
```

Comment:
This comment makes no much sense - it's plain and simple `DELETE FROM` query - there is no `DROP PARTITION` etc. in this query.

Reply:
ok, delete DROP PARTITION, ATTACH PARTITION FROM, UPDATE, DELETE, comment